library/src/main/android/permissions/dispatcher/PermissionUtils.java

```diff
@@ -94,13 +96,23 @@ public static boolean hasSelfPermissions(Context context, String... permissions)
      * @see #hasSelfPermissions(Context, String...)
      */
     private static boolean hasSelfPermission(Context context, String permission) {
+        if (Build.VERSION.SDK_INT > Build.VERSION_CODES.M && Build.MANUFACTURER.equalsIgnoreCase("Xiaomi")) {
```

Comment:
Umm, I actually I don't have Xiaomi device so can't test it...just referenced https://github.com/hotchemi/PermissionsDispatcher/issues/211#issuecomment-265660654.
@a1018875550 Any idea?